security-council-rehearsals/step-1-welcome/README.md

```diff
@@ -204,18 +204,18 @@ using keys owned by the Facilitator, to make it a M-of-2M+1 multisig.
 For example, if there are 3 participants, we recommend creating a 3 of 7
 multisig with 4 additional keys owned by the Facilitator.
 
-#### 2. Create a new task in the `sep` directory:
```

Comment:
Curious here, if there is still the instruction for `sep` since we replace all them with `eth`